test/utils/widget_test_harness.dart

```diff
@@ -1,30 +1,35 @@
+import 'package:crowdleague/actions/redux_action.dart';
 import 'package:crowdleague/models/app/app_state.dart';
 import 'package:flutter/material.dart';
 import 'package:flutter_redux/flutter_redux.dart';
-import 'package:redux/redux.dart';
 
 import '../mocks/redux/redux_store_mocks.dart';
 
 /// A test harness to wrap a widget under test and provide all the functionality
 /// that a test may want in order to interact with the widget or check for
 /// expected values and behaviour.
 class WidgetTestHarness {
-  final Store<AppState> _store;
+  final FakeStore _fakeStore;
```

Comment:
We should use the FakeStore anywhere we don't want the redux infrastructure (middleware, reducers, etc.) to be connected so that dispatched actions can interact with services and/or change state.  

I would suggest this is everywhere, with one or two exceptions (eg. navigation, which happens via reducers and app state)